### ISSUE TITLE
Logs: On page refresh and immediately clicking cancel query - 400 bad request error displayed

### ISSUE THREAD

**Issue by neha00290** [openobserve/openobserve#3942]
### Which OpenObserve functionalities are the source of the bug?

log search

### Is this a regression?

Yes

### Description

Steps
- Go to logs page and refresh
- Click on cancel query button immediately
- Observe the error displayed 

### Please provide the version you discovered this bug in (check about page for version information)

```true
Version
Commit Hash
Build Date
v0.10.8-rc4
410aff4d1e4726f080b8f42230dabc8f7314b82e
2024-07-08T08:03:25Z
```

**Comment by omkarK06** [comment]
fixed in #3705

**Comment by neha00290** [comment]
closing this as fixed
